posts/2020-09-10-wg-prio-call-for-contributors.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: A call for contributors from the WG-prioritization team
+author: The Rust WG-Prioritization Team
+release: true
+---
+
+Are you looking for opportunities to contribute to the Rust community? Have some spare time to donate? And maybe learn something interesting along the way?
+
+The [WG-prioritization][wg-prio] can be the right place for you: we are looking for new contributors!
+
+## What is the WG-prioritization?
+
+The Prioritization WG is a compiler Working Group dedicated to handling the most important bugs found in the Rust compiler (`rustc`), to ensure that they are resolved. We stand at the frontline of the [Github rust issue tracker](https://github.com/rust-lang/rust/issues) and our job is to do triaging, mainly deciding which bugs are critical (potential release blockers) and prepare the weekly agenda for the Compiler Team with the most pressing issues to be taken care of.
+
+Here is a bit more [comprehensive description][wg-prio]. How we work is detailed [on the Rust Forge](https://forge.rust-lang.org/compiler/prioritization.html).
+
+Our tooling is mainly the [triagebot](https://github.com/rust-lang/triagebot), a trustful messenger that helps us by sending notification to our [Zulip stream][zulip-wg-prio] when an issue on Github is labelled.
+
+We also have a repository with some [issues and meta-issues](https://github.com/rust-lang/compiler-team-prioritization/issues), where we basically note down how we would like our workflow to evolve. Contributions to these issues are welcome, but a bit more context about the workflow of this Working Group is probably necessary.
+
+Documentation is also a fundamental part of the onboarding package that we provide to newcomers. As we basically "organize and sort stuff", a lot happens without writing a single line of code but rather applying procedures to optimize triaging and issues prioritization.
+
+This requires our workflow to be as efficient and well documented as possible. As such, we are always open to contributions to clarify the documentation (and fresh eyeballs are especially precious for that!).
+
+## The typical week of a WG-prioritization member
+
+Our week starts on Thursday/Friday after the Rust Compiler Team meeting (one of the cool teams that keep that beast at bay). We summarize the meeting and draft an agenda for the following one.
```

Comment:
This change is technically true but I find that for the target audience of this blog post it may be confusing. Among other things I tried keeping a very high-level view of our workflow.

Reply:
That's fair but I want to get rid of the idea that we summarize the meeting because that's not true. Proposing another change ...